examples/svd/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
I have create a project [Yaserde Examples](https://github.com/media-io/yaserde_examples) to put some example for tests on bug fixes etc.
As this folder is not included in the main Cargo.toml, I'm not sure it's relevant to keep it like that.
2 options:
- keep an another repository with examples, difficult for devs to find the link ...
- move all examples into this sub-folder as you started, and include it in the main root. Like that it will be tested each time we contribute to the project.

Reply:
I'd definitely prefer the second option, I'll do that now.

Reply:
Agreed, thank you !